2024-03-24 17:01:51 INFO [32mpytest tests/frontend/test_e2e_project.py::test_e2e_create_project[firefox] (setup)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:*/*'
-H 'Connection:keep-alive'
-H 'Authorization:Basic OjM3NjM3NDk3ODAwOTg3ODM1NzE=' 
-d ''
2024-03-24 17:02:27 INFO [32mpytest tests/frontend/test_e2e_project.py::test_e2e_create_project[firefox] (teardown)[0m
curl - X DELETE 'http://localhost:8111/app/rest/projects/id:yuuNjuZZxZw' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:0fbff5c9-48df-4b0b-acc0-33c0527c4af9'
-H 'Cookie:TCSESSIONID=972EB94364501293AA945AC84358C132'
-H 'Content-Length:0'
-H 'Authorization:Basic OjM3NjM3NDk3ODAwOTg3ODM1NzE=' 
-d '' 
 

2024-03-24 17:02:27 INFO 	RESPONSE:
STATUS_CODE: [31m404[0m
DATA: [31mResponding with error, status code: 404 (Not Found).
Details: jetbrains.buildServer.server.rest.errors.NotFoundException: No project found by locator 'count:1,id:yuuNjuZZxZw'. Project cannot be found by external id 'yuuNjuZZxZw'.
Could not find the entity requested. Check the reference is correct and the user has permissions to access the entity.[0m